spec/rails_helper.rb

```diff
@@ -1,6 +1,9 @@
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 require 'simplecov'
 require 'simplecov-json'
+require 'webmock/rspec'
+
+WebMock.disable_net_connect!
```

Comment:
this ensures we don't do any real requests during tests (unless we're generating a "cassette" with VCR